source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../variables/breakpoint";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+}
+
+.figure__image {
+  @include block-spacing($start: $baselinegrid-space-extra_small, $end: $baselinegrid-space-extra_small);
+  @include margin(12px, inline);
+  border: 1px solid $color-text-dividers;
```

Comment:
Should probably use `$grid-divider_size` (maybe that needs to lose the reference to 'grid' in the name?)

Maybe should use the `divider` mixin too (and rename to just `border`)?

Reply:
Good call. Let's rename `divider()` to `border()` and use that here in preference to the bare `border` declaration.